test/python/test_statevector_simulator_sympy.py

```diff
@@ -22,15 +22,14 @@
 from sympy import sqrt
 
 from qiskit import qasm, unroll, QuantumProgram, QuantumJob
-from qiskit.backends.local.sympy_qasmsimulator import SympyQasmSimulator
+from qiskit.backends.local.statevector_simulator_sympy import StatevectorSimulatorSympy
```

Comment:
I'm commenting on this line - but can you check `test/python/test_simulator_interfaces.py`? It seems to be empty, and probably the place where the tests for the aliases were meant to be (which we actually need!).

Reply:
added tests